gradle/validation/error-prone.gradle

```diff
@@ -15,31 +15,11 @@
  * limitations under the License.
  */
 
-def skipErrorProne = rootProject.runtimeJavaVersion > JavaVersion.VERSION_15;
```

Comment:
Hi,
Error Prone still does not work with Java 16+

I have to revert those changes in this file for Jenkins to pass with JDK 16, JDK 17 and JDK 18.

Reply:
Done in https://github.com/apache/solr/commit/a793d13be5f04ed1e827f62465b414e07c59692c